.github/workflows/ci.yaml

```
@@ -0,0 +1,47 @@
+name: "CI"
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 7 * * *"
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.OS }}-latest
+    name: "tests"
+    strategy:
+      matrix:
+        PYTHON:
+          - 3.8
+          - 3.9
+        OS:
+          - ubuntu
+          - windows
+          - macos
```

I'm personally partial to matrix elements looking like this instead (just seems clearer that it's a list):

```suggestion
        PYTHON: [3.8, 3.9]
        OS: [ubuntu, windows, macos]
```

I will say I'm more partial to lower case variables (similar to: https://github.com/OpenFreeEnergy/Lomap/blob/b31e6cde3f8e4117e75fd592ac925c17b77cae71/.github/workflows/CI.yaml#L31-L34) but I can live with things either way.